hud-data/ultrahud-tf2.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "ULTRAHUD-TF2",
+  "author": "The 2fort Cremator",
+  "contributors": [
+    "Raysfire and VillageGreenPreserver"
+  ],
+  "releaseDate": "2024-02-24",
+  "resources": [
+    "ultrahud-logo"
+  ],
+  "flags": [
+    "menus",
+    "crosshairs"
+  ],
+  "repo": "https://github.com/cremator-2fort/ULTRAHUD-TF2",
+  "hash": "f9ec4efc0e673b26627e0a8cb2d301036c98a1c0",
+  "prerelease": false,
```

```suggestion
  "prerelease": true,
```

If it's early access, would you like to set prerelease to true?

---

Yeah, thats fine.